lib/Controller/RoomController.php

```diff
@@ -201,8 +201,14 @@ public function getRooms(int $noStatusUpdate = 0): DataResponse {
 	 * @return DataResponse
 	 */
 	public function getSingleRoom(string $token): DataResponse {
+		$isSIPBridgeRequest = $this->validateSIPBridgeRequest($token);
```

Comment:
If headers are provided but invalid, an error should be returned instead of returning public information.

Reply:
Gives a `401 Unauthorized` now